demos-europe/demosplan-core
# Fix(refs: T35961): incorrect redirect by click to procedure in nav menu

***Ticket: https://yaits.demos-deutschland.de/T35961***

### Description:

- there are 2 different cases due to the role:

### 1. case: the case with the role as data capturer you see a list of the captured statements of the procedure with a different path and permission
\
![screnshot24](https://github.com/demos-europe/demosplan-core/assets/104975238/6577aeb6-b9dc-44db-b94f-3f63dc427d81)

### 2. case: the other cases you see the overview of the procedure
\
![screenshot25](https://github.com/demos-europe/demosplan-core/assets/104975238/d5143a77-b674-4536-ad79-326fb97006e1)


### How to review/test
- login as data capturer, click to a procedure and than e.g. to statement list. than click back to the procedure from the nav menu, you will see the statement list
- login as fpa and do the same and repeat the same way like described


### PR Checklist
